CHANGES.rst

```diff
@@ -1,6 +1,35 @@
 Traits CHANGELOG
 ================
 
+Release 6.1.1
+-------------
+
+Released: 2020-07-23
+
+Traits 6.1.1 is a bugfix release fixing a handful of minor documentation and
+test-related issues with the Traits 6.1.0 release. There are no API-breaking
+changes in this release. It's recommended that all users of Traits 6.1.0
+upgrade to Traits 6.1.1.
```

Comment:
Is it really important for users to upgrade?

Reply:
Good point. No, not really.

But this text is already in the release changelog, so it doesn't make sense to change it here.

Reply:
Ah. OK. If we skip this sentence in the announcement that would be just as good :)

Reply:
Yep, I'll do that. Thanks!